cmd/dep/init.go

```diff
@@ -254,59 +252,46 @@ func getProjectData(ctx *dep.Ctx, pkgT gps.PackageTree, cpr string, sm *gps.Sour
 		fmt.Fprintf(os.Stderr, "%s %s\n", message, pr)
 	}
 
-	for _, v := range pkgT.Packages {
-		// TODO: Some errors maybe should not be skipped ;-)
-		if v.Err != nil {
-			vlogf("%v", v.Err)
-			continue
-		}
-		vlogf("Package %q, analyzing...", v.P.ImportPath)
-
-		for _, ip := range v.P.Imports {
-			if isStdLib(ip) {
-				continue
-			}
-			if hasImportPathPrefix(ip, cpr) {
-				// Don't analyze imports from the current project.
-				continue
-			}
-			pr, err := sm.DeduceProjectRoot(ip)
-			if err != nil {
-				return projectData{}, errors.Wrap(err, "sm.DeduceProjectRoot") // TODO: Skip and report ?
-			}
+	rm, _ := pkgT.ToReachMap(true, true, false, nil)
+	if len(rm) == 0 {
+		return projectData{}, nil
+	}
 
-			packages[ip] = true
-			if _, ok := dependencies[pr]; ok {
-				if !contains(dependencies[pr], ip) {
-					dependencies[pr] = append(dependencies[pr], ip)
-				}
+	vlogf("Building dependency graph...")
+	for _, ip := range rm.Flatten(false) {
```

Comment:
Can you give this false a meaningful name?

eg
```
const foo = false
for _, ip := range rm.Flatten(foo) {
```

because right now I have no idea what false means here.